tools/cli/internal/openapi/filter/versioning.go

```diff
@@ -21,7 +22,7 @@ import (
 	"github.com/mongodb/openapi/tools/cli/internal/apiversion"
 )
 
-// Filter: VersioningFilter is a filter that modifies the OpenAPI spec by removing operations and responses
```

Comment:
`// Filter: ` notation is used so that we can auto gen all the filters

Reply:
ah I was getting a warning in my IDE that this is not golang standard. I will revert the changes

Reply:
I reverted my changes. thanks!